tests/integration/kubernetes/gha-run.sh

```diff
@@ -345,6 +345,54 @@ function run_tests() {
 	popd
 }
 
+# Print a report about tests executed.
+#
+# Crawl over the output files found on each "reports/yyyy-mm-dd-hh:mm:ss"
+# directory.
+#
+function report_tests() {
+	local reports_dir="${kubernetes_dir}/reports"
+	local reports
+	local ok
+	local not_ok
+	local flaky
+	local status
+
+	if [ ! -d "$reports_dir" ]; then
+		info "no reports directory found: $reports_dir"
+		return
+	fi
+
+	reports=$(ls "$reports_dir")
+	for report in $reports; do
+		ok=($(find "${reports_dir}/${report}" -name "ok-*.out"))
+		not_ok=($(find "${reports_dir}/${report}" -name "not_ok-*.out"))
+		flaky=($(find "${reports_dir}/${report}" -name "flaky-*.out"))
+
+		cat <<-EOF
+		SUMMARY ($(basename "$report")):
+		 Pass:  ${#ok[*]}
+		 Fail:  ${#not_ok[*]}
+		 Flaky: ${#flaky[*]}
+		EOF
+
+		echo -e "\nSTATUSES:"
+		for out in "${not_ok[@]}" "${flaky[@]}" "${ok[@]}"; do
+			status=$(basename "$out" | cut -d '-' -f1)
+			bats=$(basename "$out" | cut -d '-' -f2- | sed 's/.out$//')
+			echo " $status $bats"
+		done
+
+		echo -e "\nOUTPUTS:"
+		for out in "${not_ok[@]}" "${flaky[@]}" "${ok[@]}"; do
```

Comment:
In case we decide to fail this case in case of a failure I'd also suggest reorder these from OK to flaky and not_ok at the end so when we are reading it bottom to top we should see the most important (not ok) first (if we keep this case passing your order works better as we'll be reviewing it top to bottom)